core/src/main/java/io/questdb/std/IntLongHashMap.java

```diff
@@ -27,19 +27,15 @@
 import java.util.Arrays;
 
 public class IntLongHashMap extends AbstractIntHashSet {
-    private static final int noEntryValue = -1;
+    private static final long noEntryValue = -1;
```

Comment:
`noEntryValue` + `inc()` combination has a bug:
```
IntLongHashMap map = IntLongHashMap();
map.inc(42); // this call will increment a value on a non-existing key
```

We either need to check the key index in `inc()` (and `valueAt()`) instead of the value or initialize the `values` array with `noEntryValue` in `clear()` and `rehash()`. `IntIntHashMap` and maybe other hash tables exhibits the same problem.